wordpress-mobile/WordPress-Android
# Jetpack focus enable more widgets on jetpack app

This PR adds the following widgets on Jetpack app and also dynamic widget previews for Android 12 (API 31) and above
- At a glance
- All time
- Today

Fixes #17143 

<img width=320 src="https://user-images.githubusercontent.com/990349/188815641-1ddd43e1-afa2-46e9-b197-0dcd7e411a0f.png" />



To test:

Test 1:
- Build and run Jetpack app
- Long press on Jetpack app icon in app launcher or on home screen
- Ensure it show now **Widgets** option in short cuts menu

Test 2:
- Tap on **Widgets** option
- Ensure it pops up a dialog with three widgets above; and QuickPress shortcut and Views this week widget

Test 3:
- Tap and hold any widget, then drop it on home screen
- Ensure it pops up  dialog with options for Site and Color selections
- Select a Site from popup
- Select a Color; light or dark from popup
- Tap **ADD WIDGET** button which should be enable after the selections
- Ensure the widget is added to home screen

Test 4
- Use a device or Emulator with Android 12 (API 31) and above
- Change the language to other than English like French, German or Arabic
- Observe now the widget previews now actually reflect the language

NOTE:
- App icon is defaulted to Jetpack now on dynamic previews since it is anyways Jetpack powered even on WP.


## Regression Notes
1. Potential unintended areas of impact
 Tested to make sure the widgets on WordPress app work as before and not affected by this on Jetpack app

2. What I did to test those areas of impact (or what existing automated tests I relied on)
 Tested WordPress app and Jetpack apps separately

3. What automated tests I added (or what prevented me from doing so)
 Existing and updated unit tests

PR submission checklist:
